### ISSUE TITLE
[ICE]: Accessing the parent contract's selector causes an error.

### ISSUE THREAD

**Issue by Subway2023** [matter-labs/era-compiler-solidity#301]
zksolc version: ZKsync v1.5.10 (LLVM build f56a67adebe2a59b0ea27f5b9230fcd40793eaf2)
solc version: 0.8.28-1.0.1

```solidity
interface A {
    function f() external;
}
contract B {
    function g() public {}
}
contract C is B {
    function h() external {}
        bytes4 public constant s2 = B.g.selector;
}
```
```
Error: Unknown exception during compilation: Dynamic exception type: std::out_of_range
std::exception::what: map::at
```

**Comment by hedgar2017** [comment]
This error is coming from `solc`.

Could you please try `--force-evmla`?

**Comment by Subway2023** [comment]
## test1
using force-evmla also causes error. The following solc_zksync is the zksync version of solc
```
zksolc --force-evmla --bin --solc solc_zksync  test.sol

Warning: `--force-evmla` flag is deprecated: please use `--codegen 'evmla'` instead.
Warning: The `codegen` setting will become mandatory in future versions of zksolc. Please set it to either `evmla` or `yul`.

Error: Unknown exception during compilation: Dynamic exception type: std::out_of_range
std::exception::what: map::at
```
## test2
original solc does not cause error
```
solc test.sol

Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
--> /mnt/sdd1/sbw/crossCompilerFuzz/testSol/test.sol

Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.8.28;"
--> /mnt/sdd1/sbw/crossCompilerFuzz/testSol/test.sol
```

**Comment by hedgar2017** [comment]
@Subway2023 thanks, I will check.

Did you pass `--bin` to solc? It could be a dry run.

**Comment by Subway2023** [comment]
Solc can compile normally and generate the bytecode.

**Comment by hedgar2017** [comment]
Will be shipped with the next release on Monday!
